README.md

```diff
@@ -58,6 +58,27 @@ curl https://raw.githubusercontent.com/dcreemer/1pass/master/1pass > /usr/local/
 chmod a+x /usr/local/bin/1pass
 ```
 
+### Bash Completion
+
+If you would like to install bash-completion for 1pass, place the `bash-completion.sh` script in
+and accessible location and then source it from your `.bash_profile`.  For example:
+
+```sh
+mkdir -p /usr/local/etc/1pass
+curl https://raw.githubusercontent.com/dcreemer/1pass/master/bash_completion.sh > /usr/local/etc/1pass/bash_completion.sh
+echo "source /usr/local/etc/1pass/bash_completion.sh" >> ~/.bash_profile
+```
+
+By default the completion script will look for fzf on your path.  If present, it will use fzf completion
+([see here](https://github.com/junegunn/fzf#fuzzy-completion-for-bash-and-zsh)). If you do not have fzf or if you turn 
+this feature off it will revert to standard bash completion behavior.  If you would like to explicitly disable
```

Comment:
Can you add a bit more text explaining that FZF completion needs to have been explicitly enabled. On macOs with homebrew, `brew install fzf` does not enable completion by default. I had to run:
`/usr/local/opt/fzf/install --completion`, and then follow the prompts to install a `~/.fzf.bash` file, which I then sourced.